components/content_settings/core/browser/brave_content_settings_policy_provider.cc

```diff
@@ -0,0 +1,144 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/content_settings/core/browser/brave_content_settings_policy_provider.h"
+
+#include "base/containers/contains.h"
+#include "base/logging.h"
+#include "brave/components/constants/pref_names.h"
+#include "components/content_settings/core/browser/content_settings_policy_provider.h"
+#include "components/content_settings/core/browser/content_settings_utils.h"
+#include "components/content_settings/core/browser/website_settings_registry.h"
+#include "components/content_settings/core/common/content_settings_pattern.h"
+#include "components/pref_registry/pref_registry_syncable.h"
+#include "components/prefs/pref_service.h"
+
+namespace {
+
+// Duplicating declaration from
+// components/content_settings/core/browser/content_settings_policy_provider.cc
+struct PrefsForManagedContentSettingsMapEntry {
+  const char* pref_name;
+  ContentSettingsType content_type;
+  ContentSetting setting;
+};
+
+constexpr PrefsForManagedContentSettingsMapEntry
+    kBravePrefsForManagedContentSettingsMap[] = {
+        {kManagedShieldsDisabledForUrls, ContentSettingsType::BRAVE_SHIELDS,
+         CONTENT_SETTING_BLOCK},
+        {kManagedShieldsEnabledForUrls, ContentSettingsType::BRAVE_SHIELDS,
+         CONTENT_SETTING_ALLOW}};
+
+constexpr const char* kBraveManagedPrefs[] = {kManagedShieldsDisabledForUrls,
+                                              kManagedShieldsEnabledForUrls};
+
+}  // namespace
+
+namespace content_settings {
+
+BravePolicyProvider::BravePolicyProvider(PrefService* prefs)
+    : PolicyProvider(prefs) {
+  ReadManagedDefaultSettings();
```

Comment:
why we are we calling these again, they're already called in the superclass

Reply:
this is not overriden function, but removed redundant`ReadManagedContentSettings` call